src/api/time/TimeAPI.js

```diff
@@ -329,6 +329,7 @@ define(['EventEmitter'], function (EventEmitter) {
 
         this.boundsVal = newBounds;
         this.emit('bounds', this.boundsVal, true);
+        this.emit('tick', timestamp);
```

Comment:
Hrm, don't want to add an extra event to the time API.  Note that the `bounds` event is fired with a `tick` argument that is `true` whenever it is coming from a clock.  You can use that to only react to ticks.